src/components/lend/LendPositions.tsx

```diff
@@ -40,6 +41,17 @@ export const LendPositions: React.FC<PoolComponentProps> = ({ poolId }) => {
               textColor={theme.palette.lend.main}
               sx={{ width: '100%', padding: '6px' }}
             ></StackedText>
+            {userPoolData.emissions.supplyEmissions && (
+              <FlameIcon
+                width={22}
+                height={22}
+                title={` This asset earns ${getEmissionsPerDayPerUnit(
+                  reserve.supplyEmissions?.config.eps || BigInt(0),
+                  reserve.estimates.supplied,
+                  reserve.config.decimals
+                ).toFixed(2)} BLND/day emissions`}
+              />
+            )}
```

Comment:
This is in the header, no?

Shouldn't these changes be on the `LendPositionCard`?

Some comment for `BorrowPositions`

Reply:
Also a note - `userPoolData` does not contain the necessary info to determine the emissions status the BLND/day emissions.

We will need to use the `Reserve` object, as done in `...MarketCard`